baseapp/baseapp.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 type (
-	execMode uint8
+	execMode = uint8
```

Comment:
I think we can revert this, as we don't need to pass execmode in the wrapped run tx